bin/garden-init

```diff
@@ -30,30 +30,35 @@ exclude=
 features=
 noMergedUsr=
 noCheckGpg=
+
+# Update all param flags
 while true; do
 	flag="$1"; shift
 	dgetopt-case "$flag"
 	case "$flag" in
-		--debian)	nonDebian= ;;
-		--debian-ports)	nonDebian= ; debianPorts=1 ;;
-		--non-debian)	nonDebian=1 ;;
-		--debootstrap)	debootstrap="$1"; shift ;;
-		--debootstrap-script) script="$1"; shift ;;
-		--keyring)	keyring="$1"; shift ;;
-		--keyring-plain) keyringPlain="$1"; shift ;;
-		--arch)		arch="$1"; shift ;;
-		--include)	include="${include:+$include,}$1"; shift ;;
-		--exclude)	exclude="${exclude:+$exclude,}$1"; shift ;;
-		--features)	features="${features:+$features,}$1"; shift ;;
-		--check-gpg)	noCheckGpg=  ;;
-		--no-check-gpg)	noCheckGpg=1 ;;
+		--debian)		nonDebian= ;;
+		--debian-ports)		nonDebian= ; debianPorts=1 ;;
+		--debian-mirror)	debianMirror=0 ; debianMirror=1 ;;
```

Comment:
Maybe simplify to just `debianMirror=1 ;;`? Maybe a typo, and intention was to set a different variable to 0?